lms/djangoapps/progress/management/commands/recalculate_progress_for_users.py

```diff
@@ -0,0 +1,103 @@
+"""
+One-time data migration script -- shoulen't need to run it again
```

Comment:
Typo: "shoulen't". And I actually think it is somewhat likely that we may need this again in the future (we've had these issues before...). Also, for posterity, I think you should add the date and either explain the issue that this fixes, or link to the PR/JIRA tickets involved.